.travis.yml

```diff
@@ -2,17 +2,9 @@ language: node_js
 node_js: node
 cache: npm
 env:
-- TEST_ENV=cjs BUILD_ENV=cjs
-- TEST_ENV=umd BUILD_ENV=umd
+- TEST_ENV=cjs
+- TEST_ENV=umd
```

Comment:
I ended up removing the specific `BUILD_ENV` configs (and just build everything). With `BUILD_ENV` specific builds, I was running into issues because the `postinstall` script (which calls triggers each package to be built) was running before the `before_script`. I don't think that this causes a significant change in the CI runtime, so it should be fine.